src/commands/branches.ts

```diff
@@ -103,6 +109,36 @@ export const builder = (argv: yargs.Argv) =>
         }),
       async (args) => await reset(args as any),
     )
+    .command(
+      'restore <id|name> <point-in-time>',
+      'Restore a branch to a point in time.\nPoint in time format is following ^self|^parent|source-branch-(id|name)[@(lsn|timestamp)]',
```

Comment:
Suggest changing to:

'restore <target-id|name> <source>[@(timestamp|lsn)]',
'Restores a branch to a specific point in time',
'<source> can be: ^self, ^parent, or <source-branch-id|name>',